CI Tests against nightly rust are failing

**Describe the bug**

The  checks started failing here:
https://github.com/apache/arrow-rs/runs/4286145388?check_suite_focus=true

```

failures:

---- src/array/mod.rs - array (line 49) stdout ----
error[E0259]: the name `arrow` is defined multiple times
 --> src/array/mod.rs:50:1
  |
2 | extern crate arrow;
  | ------------------- previous import of the extern crate `arrow` here
3 | 
4 | extern crate r#arrow;
  | ^^^^^^^^^^^^^^^^^^^^^ `arrow` reimported here
  |
  = note: `arrow` must be defined only once in the type namespace of this module

error: aborting due to previous error

For more information about this error, try `rustc --explain E0259`.
Couldn't compile the test.
---- src/datatypes/schema.rs - datatypes::schema::Schema::new (line 55) stdout ----
error[E0259]: the name `arrow` is defined multiple times
 --> src/datatypes/schema.rs:55:1
  |
2 | extern crate arrow;
  | ------------------- previous import of the extern crate `arrow` here
3 | extern crate r#arrow;
  | ^^^^^^^^^^^^^^^^^^^^^ `arrow` reimported here
  |
  = note: `arrow` must be defined only once in the type namespace of this module

error: aborting due to previous error

For more information about this error, try `rustc --explain E0259`.
Couldn't compile the test.
---- src/datatypes/schema.rs - datatypes::schema::Schema::new_with_metadata (line 72) stdout ----
error[E0259]: the name `arrow` is defined multiple times
 --> src/datatypes/schema.rs:72:1
  |
2 | extern crate arrow;
  | ------------------- previous import of the extern crate `arrow` here
3 | extern crate r#arrow;
  | ^^^^^^^^^^^^^^^^^^^^^ `arrow` reimported here
  |
  = note: `arrow` must be defined only once in the type namespace of this module

error: aborting due to previous error

For more information about this error, try `rustc --explain E0259`.
Couldn't compile the test.

failures:
    src/array/mod.rs - array (line 49)
    src/datatypes/schema.rs - datatypes::schema::Schema::new (line 55)
    src/datatypes/schema.rs - datatypes::schema::Schema::new_with_metadata (line 72)

test result: FAILED. 128 passed; 3 failed; 0 ignored; 0 measured; 0 filtered out; finished in 135.22s

error: test failed, to rerun pass '--doc'
Error: Process completed with exit code 101.
```

**To Reproduce**
Run CI on nightly


**Expected behavior**
Tests should pass

--- a/arrow/src/array/mod.rs
+++ b/arrow/src/array/mod.rs
@@ -47,8 +47,6 @@
 //!
 //! # Example
 //! ```
-//! extern crate arrow;
-//!
 //! use arrow::array::Int16Array;
 //!
 //! // Create a new builder with a capacity of 100

--- a/arrow/src/datatypes/schema.rs
+++ b/arrow/src/datatypes/schema.rs
@@ -53,7 +53,6 @@ impl Schema {
     /// # Example
     ///
     /// ```
-    /// # extern crate arrow;
     /// # use arrow::datatypes::{Field, DataType, Schema};
     /// let field_a = Field::new("a", DataType::Int64, false);
     /// let field_b = Field::new("b", DataType::Boolean, false);
@@ -70,7 +69,6 @@ impl Schema {
     /// # Example
     ///
     /// ```
-    /// # extern crate arrow;
     /// # use arrow::datatypes::{Field, DataType, Schema};
     /// # use std::collections::HashMap;
     /// let field_a = Field::new("a", DataType::Int64, false);

